src/test/librbd/cache/test_mock_ParentCacheObjectDispatch.cc

```diff
@@ -39,6 +38,39 @@ struct TypeTraits<MockParentImageCacheImageCtx> {
 
 }; // namespace cache
 
+namespace io {
+namespace util {
+
+namespace {
+
+struct Mock {
```

Comment:
Nit: where is this being instantiated?

Reply:
Currently nowhere. I needed `read_parent` defined so it could be compiled. But currently `read_parent` is never triggered by the tests (and I am not sure why, I didn't look carefully at the tests though, probably I should). So the `read_parent` could be just an empty function here but I wanted to make it mock from the start.